__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const request = require("supertest");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return connection.end();
+});
+
+describe("GET /api/topics", () => {
+  test("returns 200 status code and an array of objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(Array.isArray(response.body.topic)).toBe(true);
+        expect(response.body.topic.length).toBeGreaterThan(0);
+        expect(typeof response.body.topic[0]).toBe("object");
```

Comment:
this isn't making sure every single item in the array is an object, I would iterate over the topics here to do that. I'd also like you to make sure the keys on each object are how you'd expect them to look here too